app/src/main/java/com/sopt/now/compose/SignUpActivity.kt

```diff
@@ -0,0 +1,218 @@
+package com.sopt.now.compose
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.now.compose.Constants.Companion.MAX_ID_LENGTH
+import com.sopt.now.compose.Constants.Companion.MAX_PASSWORD_LENGTH
+import com.sopt.now.compose.Constants.Companion.MBTI_LENGTH
+import com.sopt.now.compose.Constants.Companion.MIN_ID_LENGTH
+import com.sopt.now.compose.Constants.Companion.MIN_PASSWORD_LENGTH
+import com.sopt.now.compose.Constants.Companion.USER_DATA
+import com.sopt.now.compose.ui.theme.NOWSOPTAndroidTheme
+
+class SignUpActivity : ComponentActivity() {
+    private lateinit var userData: SignUpData
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            NOWSOPTAndroidTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    showSignup(onSignupBtnClicked = { id, password, nickname, mbti ->
+                        userData = SignUpData(id, password, nickname, mbti)
```

Comment:
```suggestion
                    showSignup { signUpData ->
                        userData = signUpData
```
이렇게 바꿀 수도 있을 것 같습니다!